docs/source/index.md

```diff
@@ -15,7 +15,7 @@ Although optimized for Scylla, the driver is also compatible with [Apache Cassan
 * [Quick start](quickstart/quickstart.md) - Setting up a Rust project using `scylla-rust-driver` and running a few queries
 * [Migration guides](migration-guides/migration-guides.md) - How to update the code that used an older version of this driver
 * [Connecting to the cluster](connecting/connecting.md) - Configuring a connection to scylla cluster
-* [Making queries](queries/queries.md) - Making different types of queries (simple, prepared, batch, paged)
+* [Making queries](queries/queries.md) - Making different types of queries (unprepared, prepared, batch, paged)
```

Comment:
Let's ditch the notion of _query_ from the driver and use _statement_ exclusively. Here, I would replace _making queries_ with _creating and executing statements_.

Reply:
That would require a change in naming (replacing `Query` with `UnpreparedStatement`), otherwise you would have a `Query` type but not of the docs would mention queries. What do yuo think about splitting in in 2 PRs - in this one let's improve prepared statement docs, and in the next one let's replace Query with UnpreparedStatement (and fix docs), which would fix https://github.com/scylladb/scylla-rust-driver/issues/713

Reply:
Sounds good to me, I can take care of that.